### PR TITLE
scripts/test.sh: remove TLDR005 from ignore list for languages

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

## Changes

- Remove TLDR005 from the `tldr-lint` ignore list for languages.
- Minor fixes in comments.

----
This is what the error code does (we use colons across all languages [except Chinese pages] for example descriptions so this rule shouldn't be ignored):

<html>
<body>

TLDR005 | Example descriptions should end in a colon with no trailing characters
-- | --

</body>
</html>

### CODE REVIEW

**Comment by kbdharun** [comment]
I guess we can ignore `zh` and `zh-TW` pages alone from the script.

File: scripts/test.sh [hunk 1]
```diff
@@ -57,12 +57,16 @@ function run_flake8 {
...
 }
 
-# Default test function, ran by `npm test`.
+# Default test function, run by `npm test`.
 function run_tests {
   find pages* -name '*.md' -exec markdownlint {} +
   tldr-lint ./pages
   for f in ./pages.*; do
-    tldr-lint --ignore "TLDR003,TLDR004,TLDR005,TLDR015,TLDR104" "${f}"
+    if [[ $f == *zh* || $f == *zh_TW* ]]; then
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/test.sh:65]: I think it is better to leave it as a wildcard here so that new languages aren't ignored (we can exclude them if it adds a trailing space like Chinese pages).

[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/test.sh:65]: Yep

[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/test.sh:65]: Don't they have special characters, i.e in languages like Spanish/French the first character might be a special one like ṗ not excluding the first character should be a capital one would be problomatic.

[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/test.sh:65]: > But that's only TLDR003. TLDR004, TLDR015, TLDR104 are also ignored for too many languages.

I think TLDR004 detects more info link line too (we need to remove it first before adding here as we are discussing removing periods behind more information link line).

TLDR015 is same as TLDR003 with only difference being it works on example descriptions.

TLDR104 on the other hand was specifically handcrafted for the English page in https://github.com/tldr-pages/tldr-lint/commit/a9b22cc6905c27d9abba8b363300c0ff6c869de1, we need to rework it as a whole if we need to extend it to other languages I think.

[the remark this answers is not in the document]
Comment (kbdharun) [reply scripts/test.sh:65]: I will unresolved this conversation for historical reference, to look up easily in future.

**Review by gutjuri — APPROVED** [review]
